packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -3234,7 +3238,10 @@ class _InspectorOverlayLayer extends Layer {
     Rect targetRect,
   ) {
     canvas.save();
-    final double maxWidth = size.width - 2 * (_kScreenEdgeMargin + _kTooltipPadding);
+    final double maxWidth = math.max(
+      size.width - 2 * (_kScreenEdgeMargin + _kTooltipPadding),
+      0,
```

Comment:
good catch! should we just short circuit and not try to draw anything is maxWidth <= 0?

Reply:
Perhaps we could but I don't know enough about these cases to know if we always want to.